core/common/src/main/java/alluxio/network/protocol/RPCFileReadResponse.java

```diff
@@ -85,7 +85,7 @@ public static RPCFileReadResponse decode(ByteBuf in) {
     short status = in.readShort();
 
     DataBuffer data = null;
-    if (length > 0) {
+    if (length >= 0) {
```

Comment:
I see, usually adding a comment for this corner case handling would be good for readability.